lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+# require './lib/takeaway.rb'
+
+class Menu
+  attr_reader :dishes
+  def initialize
+    @dishes = {
+      "marinara" => 5,
+      "margherita" => 6,
+      "napoli" => 6,
+      "calzone" => 8
+    }
+  end
+
+  def show_menu
```

Comment:
You can try putting this sort of thing in a "presentation class" which will take the menu data and format it to look nice (which I didn't do either, but I heard it was useful)